packages/pwa-kit-create-app/scripts/create-mobify-app.js

```diff
@@ -222,25 +224,23 @@ const retailReactAppPrompts = () => {
 
     const questions = [
         {
-            name: 'projectId',
-            validate: validProjectId,
-            message: 'What is your project ID (example-project) in Managed Runtime Admin?'
+            name: 'projectName',
+            validate: validProjectName,
+            message: 'What is the name of your Project?'
         },
         {
             name: 'instanceUrl',
-            message:
-                'What is the URL (https://example_instance_id.sandbox.us01.dx.commercecloud.salesforce.com) for your B2C Commerce instance?',
+            message: 'What is the URL for your Commerce Cloud instance?',
             validate: validUrl
         },
         {
             name: 'clientId',
-            message: 'What is your API client ID?',
+            message: 'What is your Commerce API client ID in Account Manager?',
```

Comment:
This is not the right question.

We need to know your SLAS Client ID.

Reply:
This is the new copy that @jennmills sent me. 
I'll ping her to adjust the copy to reflect SLAS Client ID.